packages/core/index.js

```diff
@@ -3,12 +3,19 @@ const fs = require("fs");
 const path = require("path");
 const { getWeb3 } = require("@uma/common");
 
+function getDirectoryForVersion(version) {
```

Comment:
how does this function handle if there is no `package.json` for the associated version specified? this function is called by `getArtifact` which is called in a few places such as `getAbi`. what if I put a version that is not defined in the base `package.json`?

Reply:
Nvm I see that you test this in the mocha tests.

Reply:
Just for posterity, if the version is bad OR the file name is bad, it just throws.